### PR TITLE
Azure CPlat PR : Update NextLink Definition for VM.List(rgName)

### PR DESCRIPTION
**Pull request by huangpf**

### CODE REVIEW

**Comment by huangpf** [comment]
@amarzavery Could you please help review? Thanks.

**Comment by huangpf** [comment]
https://github.com/Azure/azure-sdk-for-node/pull/1785

**Comment by huangpf** [comment]
@amarzavery It doesn't seem like the spec change has any effects on ```nextLink```?

File: arm-compute/2016-03-30/swagger/compute.json [hunk 1]
```diff
@@ -1171,7 +1171,7 @@
           }
         },
         "x-ms-pageable": {
-          "nextLinkName": null
+          "nextLinkName": "nextLink"
         }
       }
```
Comment (John-Hart) [inline arm-compute/2016-03-30/swagger/compute.json]: This change looks fine but I see that @hyonholee has made the same change in  PR #359 and is also trying to fix #352 to fix the broken CI.  If he's successful this PR can be cancelled

**Pull request closed without merging**